api/src/test/kotlin/edu/wgu/osmt/elasticsearch/SearchControllerTest.kt

```diff
@@ -113,4 +121,33 @@ internal class SearchControllerTest @Autowired constructor(
         // Assert
         assertThat(result.body?.map { it.name }).contains(listOfKeywords[0].value)
     }
-}
\ No newline at end of file
+
+    @Test
+    fun similarSkillWarningsShouldFindSimilarities() {
+        richSkillRepository.createFromApi(
+            listOf(ApiSkillUpdate(
+                    "Access and Security Levels Standardization",
+                    "Standardize levels of access and security to maintain information security.",
+                    PublishStatus.Draft
+                )),
+            "admin",
+            "admin@wgu.edu"
+        )
+        val response = searchController.similarSkillWarnings(
+            arrayOf(ApiSimilaritySearch("Standardize levels of access and security to maintain information security."))
+        )
+        assertThat((response as ResponseEntity).statusCode).isEqualTo(HttpStatus.OK)
+        assertThat(response.body?.size).isEqualTo(1)
+        assertThat(response.body?.first()?.size).isEqualTo(1)
+    }
+
+    @Test
+    fun similarSkillWarningsShouldNotFindSimilarities() {
+        val response = searchController.similarSkillWarnings(
+            arrayOf(ApiSimilaritySearch("Access an application programming interface (API) with a programming language to change data for a task."))
+        )
+        assertThat((response as ResponseEntity).statusCode).isEqualTo(HttpStatus.OK)
+        assertThat(response.body?.size).isEqualTo(1)
```

Comment:
same here, we don't need this one since it is asserted in the question mark (.first()?) in line 151.